client/src/containers/App/index.js

```diff
@@ -18,12 +18,17 @@ class AppContainer extends Component {
     <App
       todoCards={this.state.todoCards}
       addTodoCard={this.addTodoCard}
+      deleteTodoCard={this.deleteTodoCard}
     />
   )
 
   addTodoCard = (card) => {
     this.setState(state => ({ todoCards: [...state.todoCards, card] }))
   }
+
+  deleteTodoCard = (cardId) => {
```

Comment:
@AleksSenkou linter doesn't support it in this case 🤔

Reply:
@DDKatch can you give an error example and other options?

Reply:
@DDKatch I guess it is `Expected parentheses around arrow function argument having a body with curly braces` that seems ok because the body is rounded with `{}`

Reply:
I'm going to fix it in https://trello.com/c/xkkmIu2M/40-when-i-develop-the-app-id-like-not-to-see-linter-errors

Reply:
@AleksSenkou let's merge this pr after yours because of that

Reply:
@AleksSenkou I've rebased it on the new master